cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -77,7 +77,7 @@ spec:
             path: /readiness
             port: 8081
             scheme: HTTP
-          # we poll on pod startup for the Kubernetes master service and
+          # we poll on pod startup for the Kubernetes control-plane service and
```

Comment:
@vanduc95 
on lines 176 and 244 in this file, MasterTaintKey has to be modified too.

this e2e test caught that:
https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/74473/pull-kubernetes-e2e-kind/5/build-log.txt

> phase addon/coredns: error when parsing CoreDNS deployment template: error when executing template: template: template:28:16: executing \"template\" at <.MasterTaintKey>: can't evaluate field MasterTaintKey in type struct { DeploymentName string; Image string; ControlPlaneTaintKey string }"

Reply:
/hold

Reply:
Thanks @neolit123, I have updated.